### PR TITLE
telegram alert for e2e tests

### PR DESCRIPTION
**Pull request by pandey-shubh**
* alerts to be sent on telegram channel when test fails
* bot is added to telegram channel : https://t.me/openloginalert
* bot is promoted to admin and using its API key, message is sent to channel

### CODE REVIEW

File: .github/workflows/openlogin.existing-v2-user-login.yml [hunk 1]
```diff
@@ -62,6 +62,11 @@ jobs:
         if: steps.download-state.outcome == 'success'
         run: yarn run playwright test --config openlogin/existing-v2-user-login/index.config.ts --project ${{ matrix.browser }}
 
+      - id: telegram-alert
+        name: Alert to telegram channel
+        if: steps.run-tests.outcome != 'success'
+        run: curl "https://api.telegram.org/bot5392115831:AAEThkvZV6doDUKIPM6oX03sHGHAcpVwwL4/sendMessage?chat_id=-1001742900419&text=Existingv2UserLogin%0ATestresult=Failed%0AOS=${{matrix.os}}%0Abrowser=${{matrix.browser}}"
```
Comment (tainguyentt) [inline .github/workflows/openlogin.existing-v2-user-login.yml:68]: This is public repo, should we expose credentials here?

Comment (pandey-shubh) [reply .github/workflows/openlogin.existing-v2-user-login.yml]: updated

File: .github/workflows/openlogin.homepage.yml [hunk 2]
```diff
@@ -52,6 +52,11 @@ jobs:
         if: steps.yarn-install.outcome == 'success' && steps.yarn-install-deps.outcome != 'failure'
         run: yarn run playwright test --config openlogin/homepage/index.config.ts --project ${{ matrix.browser }}
 
+      - id: telegram-alert
+        name: Alert to telegram channel
+        if: steps.run-tests.outcome != 'success'
+        run: curl "https://api.telegram.org/bot${{secrets.BOT_API_TOKEN}}/sendMessage?chat_id=${{secrets.CHAT_ID}}&text=Homepage%0ATestresult=Failed%0AOS=${{matrix.os}}%0Abrowser=${{matrix.browser}}"
```
Comment (tainguyentt) [inline .github/workflows/openlogin.homepage.yml:58]: should we set `continue-on-error: true`?

Comment (pandey-shubh) [reply .github/workflows/openlogin.homepage.yml:58]: added

**Review by tainguyentt — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
Lgtm

**Pull request merged**